cfme/infrastructure/host.py

```diff
@@ -65,7 +68,7 @@ class Host(BaseEntity, Updateable, Pretty, PolicyProfileAssignable, Taggable):
     host_platform = attr.ib(default=None)
     ipmi_address = attr.ib(default=None)
     mac_address = attr.ib(default=None)
-    credentials = attr.ib(default=None)
+    credentials = attr.ib(default=attr.Factory(dict))
```

Comment:
Question: Will this affect some other test cases?

Reply:
none of the test instantiate host with full list of attributes. we usually tend to use host methods `from_config`, which will usee host credentials from the yamls as `default`

Reply:
or we could just store creds in appropriate format in yamls